### DISCUSSION TITLE
New emoji for "like"

### DISCUSSION THREAD

**Discussion by alexec** [community/community#17317] [Other Feature Feedback, Questions, & Ideas]
I'd like to tell someone that I like their comment. Today, I can use

* 👍 - this means "acknowledged" not "I like"
* ❤️ - this mean "I love" not "I like"

I want something inbetween.

**Comment by notpushkin** [comment]
I use 👀 as “acknowledged”, most of the time.

**Reply by alexec** [reply]
Ah - I think I use 👀 to mean “I’m looking at this” but I think other people use it to mean “I’m surprised by this”. Maybe the world needs a “dictionary of emojis”?

**Reply by alexec** [reply]
https://emojipedia.org/eyes/

**Reply by alexec** [reply]
https://www.dictionary.com/e/emoji/eyes-emoji/

**Reply by alexec** [reply]
I think we may both be wrong. Who knew?

**Reply by notpushkin** [reply]
I think it's only natural that different people use it in different ways, yeah :-)

**Comment by queenofcorgis** [comment]
Do you have something in mind for what would be in-between? Any other users have ideas? 

Thanks for starting this conversation 💬

**Comment by alexec** [comment]
I'm wondering if 👍 should be "like" (`:like:`) and we actually need something else for "acknowledge", maybe "ok" 👌 ?
